apps/web/src/components/OccupationSelector.tsx

```diff
@@ -50,8 +50,19 @@ export const OccupationSelector = ({ searchValue = '', showDescriptionModal = fa
             filteredOccupations.every(o => values.occupation.includes(o.id))
           }
         />
-        <label className='font-bold' htmlFor={'selectAll'}>
+        <label className='' htmlFor={'selectAll'}>
           Select all
+          <span className='pl-1 font-bold'>
+            (
+            {
+              (
+                filteredOccupations?.filter((o: any) =>
```

Comment:
Nit - Do you have a type for this?

Reply:
Good catch! I wonder why I added any here. Type is inplicitly inferred "OccupationItemProps". Updated the code to reflect that.